external/llvm-project/mlir/lib/ExecutionEngine/RocmRuntimeWrappers.cpp

```diff
@@ -141,11 +140,6 @@ extern "C" void mgpuMemcpy(void *dst, void *src, size_t sizeBytes,
       hipMemcpyAsync(dst, src, sizeBytes, hipMemcpyDefault, stream));
 }
 
-extern "C" void mgpuMemset32(void *dst, int value, size_t count,
-                             hipStream_t stream) {
```

Comment:
Please check to see if this is used or referenced upstream - there are other people who rely on these RocmRuntimeWrappers